### PR TITLE
FUSETOOLS2-700 - use npm CI instead of npm install

### PR DESCRIPTION
**Pull request by apupier**
it allows to ensure that the package-lock.json is respected

### CODE REVIEW

**Comment by apupier** [comment]
job checking on Jenkins https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/VS%20Code/job/vscode-temporary-testing/69/

**Review by bfitzpat — APPROVED** [review]
what does "ci" do vs "install"? clean and install?

**Comment by apupier** [comment]
> what does "ci" do vs "install"? clean and install?

https://docs.npmjs.com/cli/ci#description

```
This command is similar to npm-install, except it’s meant to be used in automated environments such as test platforms, continuous integration, and deployment – or any situation where you want to make sure you’re doing a clean install of your dependencies. It can be significantly faster than a regular npm install by skipping certain user-oriented features. It is also more strict than a regular install, which can help catch errors or inconsistencies caused by the incrementally-installed local environments of most npm users.

In short, the main differences between using npm install and npm ci are:

- The project must have an existing package-lock.json or npm-shrinkwrap.json.
- If dependencies in the package lock do not match those in package.json, npm ci will exit with an error, instead of updating the package lock.
- npm ci can only install entire projects at a time: individual dependencies cannot be added with this command.
- If a node_modules is already present, it will be automatically removed before npm ci begins its install.
- It will never write to package.json or any of the package-locks: installs are essentially frozen.
```

**Review by djelinek — APPROVED** [review]
